runtime/doc/mbyte.txt

```diff
@@ -89,8 +89,8 @@ See |encoding-values| for a list of acceptable values.
 
 The result is that all the text that is used inside Vim will be in this
 encoding.  Not only the text in the buffers, but also in registers, variables,
-etc.  This also means that changing the value of 'encoding' makes the existing
-text invalid!  The text doesn't change, but it will be displayed wrong.
+etc. 'encoding' cannot be changed after initialization, as changing the value
```

Comment:
if "initialization" corresponds to that documented under "3. Initialization" in starting.txt, perhaps it should be linked to.